Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -55,6 +88,15 @@
 PROFILING_DUMP_ROWS_LIMIT = 20
 MAX_SAMPLES = 10
 
+
+class FileUploadParams(TypedDict):
+    filename: str
+    file: str
```

Comment:
what does `file` represent? the path? the content?

Reply:
* `file` is the file path
* `filename` is the name of the file

I agree that the naming is unclear, but I had to keep it consistent with the `slack_sdk.WebClient.file_upload_v2` kwargs (here is an [example usage](https://github.com/slackapi/python-slack-sdk/releases/tag/v3.19.0) from one of the Slack devs if you are interested)

@dorschw Perhaps I should add a comment next to each field to explain the meaning?

Reply:
a comment will be great, yes

Reply:
Added comments in commit 2d3ae24d9b52ec8d660e15df52420ed82c545827